sit/check_sit.py

```diff
@@ -7,11 +7,12 @@
 
 class CheckSIT(object):
 
-    def __init__(self, configs_directory='configs', session=None):
-        self.SIT_HELPER = SITHelper(configs_directory)
-        self.ROLES = self.SIT_HELPER.get_roles()
-        self.SIT_CONFIGS = self.SIT_HELPER.get_configs('sit')
-        self.TROPOSPHERE_CONFIGS = self.SIT_HELPER.get_configs('troposphere')
+    def __init__(self, configs_directory=None, session=None):
+        self.configs_directory = configs_directory
```

Comment:
the configs_directory is not used outside the constructor. This line can be removed.